### ISSUE TITLE
Accessibility scan action is failing, please fix.

### ISSUE THREAD

**Issue by seanmac5291** [seanmac5291/NCAA-D1-Softball-WebApp#8]
Actions job is failing, please review and ensure everything is set up correctly to run. Below are the errors i'm seeing:

Run npm ci
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE package: 'react-router@7.6.0',
npm warn EBADENGINE required: { node: '>=20.0.0' },
npm warn EBADENGINE current: { node: 'v18.20.8', npm: '10.8.2' }
npm warn EBADENGINE }
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE package: 'react-router-dom@7.6.0',
npm warn EBADENGINE required: { node: '>=20.0.0' },
npm warn EBADENGINE current: { node: 'v18.20.8', npm: '10.8.2' }
npm warn EBADENGINE }
